### PR TITLE
fix: add support for 'zip'

### PR DESCRIPTION
**Pull request by efimk-lu**

### CODE REVIEW

**Comment by efimk-lu** [comment]
Still missing tests

File: src/index.js [hunk 1]
```diff
@@ -86,12 +86,14 @@ class LumigoPlugin {
...
-
+			const { isZip } = await this.getPythonPluginConfiguration();
+			this.verboseLog(`Python plugin zip status ${isZip}`);
 			for (const func of functions) {
 				const handler = await this.createWrappedPythonFunction(
 					func,
 					token,
-					parameters
+					parameters,
+					isZip
```
Comment (theburningmonk) [inline src/index.js]: can we move `isZip` into `parameters`? would that make sense?

Comment (efimk-lu) [reply src/index.js]: No, the first are parameters that we pass to the tracer itself, isZip is for the way we wrap the code

**Pull request merged**